docs/readmes/proposals/p015_tech_debt_week.md

```diff
@@ -0,0 +1,190 @@
+- Page: email the TDW On-call at tdw@magma.pagerduty.com
+  - Provide a preferred contact (e.g. slack id, github id, ...)
+
+### TDW On-Call Response time
+
+During TDW, We ask that the following interactions and actions are given `highest work priority` - ideally with a response time of `< 1 hour`.  This is to ensure maximum velocity for our external contributors during this tight week work window - and to ensure we are efficient with their donated time.
+
+### TDW On-Call Responsibilities
+
+1. Monitor for communications
+   1. Pop open the video conference
+      1. If empty you can mute yourself and turn off video
+      2. Make sure sound is on!
+   2. GH Taggs of your user (consider [slack integration](https://slack.github.com/))
+   3. Slack channel [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN)
+   4. Pages
+2. General Assistance
+   1. You may be contacted for help with CI failures
+   2. You may be contacted for help with the final Merge
+3. Contribution life cycle for TDW On-Call
+   1. Find appropriate task for contributor
+      1. Area of expertise
+      2. Complexity
+      3. Compare with task scope and provided detail
+   2. Reach out to potential [Magma Issue Lead](#magma-issue-lead) and confirm ownership
+      1. With confirmation, reply to GH Issue with text `Issue Lead: <username tag>`
+      2. Update status in `TDW Available Work` coordinating GH Issue
+         1. Unassigned -> Assigned and add tag to GH user assigned
+   3. Put contributor in touch with GH Issue lead
+      1. Generally the person filing the issue
+      2. Otherwise a CODEOWNER
+      3. Wait for notifications from GH Issue Lead
+         1. E.g. Completed or MIA
+      4. Update status in `TDW Available Work` coordinating GH Issue
+         1. Assigned -> Merged
+4. Handle MIA contributions
+   1. If a [Magma Issue Lead](#magma-issue-lead-responsibilities) reaches out about lack of contact with a contributor
+      1. Attempt direct outreach for 2 days (on top of 2 days issue lead waited)
+      2. Move GH Issue back to available stack
+         1. Assigned -> Unassigned
+
+## Magma Issue Lead
+
+A Magma Issue Lead is the designated individual who has filed a GH Issue deemed appropriate for TDW inclusion.  These individuals may be a CODEOWNER for the relevant code base, but at least have a clear understanding of the issue's resolution and has strong communications channels with the CODEOWNERS implicated.  Determining who is Issue Lead will occur between TDW On-Call and relevant Mamga core team members.
+
+### Issues Lead Response time
+
+During TDW, We ask that the following interactions and actions are given `highest work priority` - ideally with a response time of `< 1 hour`.  This is to ensure maximum velocity for our external contributors during this tight week work window - and to ensure we are efficient with r donated time.
```

Comment:
with <??> donated time? The last clause is probably not what you intended?

Reply:
Trimmed!